src/server/pfs/cmds/cmds.go

```diff
@@ -658,6 +662,9 @@ want to consider using commit IDs directly.
 			var path string
 			if len(args) == 3 {
 				path = args[2]
+				if _, err := url.Parse(path); err == nil {
+					return fmt.Errorf("path \"%s\" looks like a URL; to put data from a URL, use -f <url>", path)
```

Comment:
What if I actually do want to have a filename that looks like a URL though? The only reason I think this might be a real use case is people sometimes do that with scrapes. Wget downloads sites into directories named for the URL for example. I also think this error doesn't protect them from that confusing of a case. People would presumably hit this if they do:

```
pachctl put-file repo branch s3://my_bucket
```

which should print out `reading from stdin` which hopefully makes it clear this isn't doing what they want.

Reply:
See slack for context, but I converted this to a warning. WDYT?